dookug-model/src/main/java/hu/icellmobilsoft/dookug/common/model/template/Document.java

```diff
@@ -0,0 +1,308 @@
+/*-
+ * #%L
+ * DookuG
+ * %%
+ * Copyright (C) 2023 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.dookug.common.model.template;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import hu.icellmobilsoft.coffee.model.base.javatime.AbstractIdentifiedAuditEntity;
+import hu.icellmobilsoft.dookug.common.model.template.enums.DocumentStatus;
+
+/**
+ * Table entity of documents Print means the summary of templates used during document generation.
+ *
+ * @author laszlo.padar
+ *
+ * @since 0.1.0
+ */
+@Entity
+@Table(name = "DOCUMENT")
+public class Document extends AbstractIdentifiedAuditEntity {
+
+    /**
+     * serial
+     */
+    public static final long serialVersionUID = 143238L;
+
+    /**
+     * ID of template
+     */
+    @Size(max = 30)
```

Comment:
`@NotNull`?

Reply:
copypaste, az ilyen dolgokat kesobb javitom majd :)

Reply:
bar nem is latom, hogy annak miert kellene requirednek lennie